### PR TITLE
fix(outputs.warp10): Support Infinity/-Infinity/NaN values

### PR DESCRIPTION
**Pull request by KannarFr**
- [x] Wrote appropriate unit tests.

### CODE REVIEW

**Comment by KannarFr** [comment]
!signed-cla

File: plugins/outputs/warp10/warp10.go [hunk 1]
```diff
@@ -198,8 +198,42 @@ func boolToString(inputBool bool) string {
...
+	case math.NaN():
+		retv = "NaN"
+	case math.Inf(-1):
+		retv = "-Infinity"
+	case math.Inf(1):
+		retv = "Infinity"
+	default:
+		retv = strconv.FormatFloat(inputNum, 'f', 6, 64)
+	}
+	return retv
```
Comment (srebhan) [inline plugins/outputs/warp10/warp10.go:236]: Please check the linter issue (it's valid). It says: "no number is equal to `NaN` not even `NaN` itself. So please use the correct functions here (https://pkg.go.dev/math#IsNaN):
```suggestion
	switch {
	case math.IsNaN(inputNum):
		return "NaN"
	case math.IsInf(inputNum, -1):
		return "-Infinity"
	case math.IsInf(inputNum, 1):
		return "Infinity"
	}
	return strconv.FormatFloat(inputNum, 'f', 6, 64)
```

**Review by srebhan — COMMENTED** [review]
Thanks for the nice fix @KannarFr! There is one issue in the code (see my comment below)...

**Review by srebhan — APPROVED** [review]
Beautiful. Thanks @KannarFr!

**Review by powersj — APPROVED** [review]
Thanks for the PR + tests!

**Pull request merged**